MAUI/DigitalGauge/Character-Display-Types.md

```diff
@@ -34,6 +34,9 @@ Numbers can be displayed in a digital gauge in different formats using the `Char
 
 {% endtabs %}
 
+![Display Number](Images\DigitalGauge9.png)
```

Comment:
DigitalGauge9. - name properly related feature for SEO

Reply:
Updated Changes